grunt/config/replace.js

```diff
@@ -62,7 +62,18 @@ module.exports = function (grunt, options) {
           grunt.log.writeln('WARNING: xAPI activityID has not been set');
         }
       }
-      
+
+      // Shim to preserve the 'adapt_scorm' identifier.
+      if (configJson.hasOwnProperty('_spoor')) {
+        if (configJson._spoor._advancedSettings) {
+          if (!configJson._spoor._advancedSettings._manifestIdentifier) {
+            configJson._spoor._advancedSettings._manifestIdentifier = 'adapt_scorm';
+          }
+        } else {
+          configJson._spoor._advancedSettings._manifestIdentifier = 'adapt_scorm';
```

Comment:
this line will always throw an error as cannot assign `_manifestIdentifier` to undefined `configJson._spoor._advancedSettings`

Reply:
perhaps something like this would keep the if statements to a minimum?
```js
      var spoor = configJson._spoor;
      if (spoor) {
       spoor._advancedSettings = spoor._advancedSettings || {};
        _.defaults(spoor._advancedSettings, {
            _manifestIdentifier: 'adapt_scorm'
        });
      }
```
or:
```js
      var spoor = configJson._spoor;
      if (spoor) {
       spoor._advancedSettings = spoor._advancedSettings || {};
       spoor._advancedSettings._manifestIdentifier = spoor._advancedSettings._manifestIdentifier || 'adapt_scorm';
      }
```

Reply:
That's actually much cleaner.